examples/sql_bank/main.go

```diff
@@ -54,7 +54,6 @@ func moveMoney(db *sql.DB) {
 			log.Fatal(err)
 		}
 		startTime := time.Now()
-		// Query is very slow and will be fixed by https://github.com/cockroachdb/cockroach/issues/2140
```

Comment:
The query is usually no longer very slow, but I still sometimes see log messages like:

`I0828 15:07:06.299984 34037 examples/sql_bank/main.go:61  SELECT id, balance FROM accounts WHERE id IN (130, 204) took 3.059484793s`